src/App.js

```diff
@@ -67,16 +65,26 @@ function App() {
   };
 
   return (
-    <>
-      <h1>Todo List</h1>
-        <AddTodoForm onAddTodo={addTodo} />
-      <hr />
-      {isLoading ? (
-        <p>Page Is Loading....</p>
-      ) : (
-        <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
-      )}
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <>
+              <h1>Todo List</h1>
+              <AddTodoForm onAddTodo={addTodo} />
+              <hr />
+              {isLoading ? (
+                <p>Page Is Loading....</p>
+              ) : (
+                <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
+              )}
+            </>
+          }
+        ></Route>
+        <Route path="/new" Component={<h1>New Todo List</h1>}></Route>
```

Comment:
The `<Route/>` component is self-closing so this can be cleaned up a bit like this:

```suggestion
        <Route
          path="/"
          element={
            <>
              <h1>Todo List</h1>
              <AddTodoForm onAddTodo={addTodo} />
              <hr />
              {isLoading ? (
                <p>Page Is Loading....</p>
              ) : (
                <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
              )}
            </>
          }
        />
        <Route path="/new" Component={<h1>New Todo List</h1>}/>
```